src/GRA.Web/Areas/MissionControl/Views/PerformerManagement/_PerformerManagementPartial.cshtml

```diff
@@ -2,8 +2,8 @@
 
 @if (Model.PerformerSchedulingEnabled)
 {
-    <div class="row my-4">
-        <div class="col-12 col-md-11 mb-2">
+    <div class="row my-4 text-nowrap">
+        <div class="col-9 mb-2">
```

Comment:
There's a `text-nowrap` on the parent `<div class="row...">` that shouldn't be there. That's probably why you have a `col-9` here, at XS this should probably be `col-12` - Bootstrap columns should always add up to 12. Based on the fact that the next column down is `col-12 col-sm-2` you might want this to be `col-12 col-sm-10`. I don't know if `col-sm-2` is appropriate since we added a button, you need to test at the different breakpoints by resizing the window and ensuring that the reflow makes sense. My guess is what you'll want will be closer to `col-12 col-md-9` on the first `<div>` and `col-12 col-md-3` on the second.